src/components/storefront/Navbar/Navbar.js

```diff
@@ -1,11 +1,29 @@
 import * as React from 'react';
 import { useLocation } from 'react-router-dom';
-import { Box, Divider, Grid, GridItem, Heading, HStack, Image, Link, chakra } from '@chakra-ui/react';
+import {
+  Box,
+  Divider,
+  Grid,
+  GridItem,
+  Heading,
+  HStack,
+  Image,
+  Link,
+  useBreakpointValue,
+  chakra,
+} from '@chakra-ui/react';
 import BagIconWithIndicator from 'components/storefront/BagIcon/BagIcon';
 import PreserveQueryParamsLink from 'components/storefront/PreserveQueryParamsLink/PreserveQueryParamsLink';
+import { PageContainer } from 'components/storefront/PageContainer/PageContainer';
 import Logo from 'assets/images/logoWhite.png';
 import { TeamBanner } from '../';
 
+export const NAVBAR_HEIGHT_RESPONSIVE = { base: '90px', sm: '100px', lg: '126px' };
```

Comment:
Random q but wondering if it's better style to have all of these values stored in a constants file and then imported, I feel like we may be reusing some of the same pixel values pretty often

Reply:
i think it makes sense to export them on a component level, just my 2 cents